.travis.yml

```diff
@@ -7,9 +7,9 @@ env:
   matrix:
     - TEST=javascript
     # need node installs for app_manager.XpathValidatorTest
-    - TEST=python NOSE_DIVIDED_WE_RUN=05 JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=6a JS_SETUP=yes
-    - TEST=python NOSE_DIVIDED_WE_RUN=bf JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=03 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=69 JS_SETUP=yes
+    - TEST=python NOSE_DIVIDED_WE_RUN=ba JS_SETUP=yes
```

Comment:
Hmm. yeah that does seem like a lot of tests on 3, but I suppose it matters more about the speed than the number. Thanks for the explanation. For some reason I thought they were hex values 

> As is, you'd be skipping 46 (that is, any test whose "name" MD5 starts with 4, 5, or 6) and probably af.

The goal was to make the tests time out less :)